Clarify how to update SDK version on Linux when using SSI (#31546)
DataDog/documentation: content/en/tracing/trace_collection/automatic_instrumentation/single-step-apm/linux.md

## Set SDK tracer versions

By default, Single Step Instrumentation installs the latest versions of Datadog APM SDKs.

> You may want to choose specific SDK versions.

Reviewer: What was your reasoning behind making this change? I think the information provided by the previous version is relevant and worth including.

Author: I've reinstated the reasons part - when I read it I thought it was implying that you could customize versions depending on the language version, but having re-read it I think it's giving reasons why you want to choose a specific version.

I'm no sure the second sentence is adding anything, but I can restore it if you think it's useful?

Reviewer: Thanks, the latest version looks reasonable.

   {{< img src="tracing/trace_collection/apm-instrumentation-version-pinning.png" alt="The 'Customize library versions' drop-down in the instructions for installing the Datadog Agent on Linux" style="width:100%;" >}}

1. Find your language(s) and use the dropdown to either:
   - Select an exact SDK version, or

>    - Select the major version which uses the latest minor release available when the Agent installation command is run.

Reviewer: ```suggestion
   - Select the major version, which uses the latest minor release available when the Agent installation command is run.
```

>    - Select the major version which will use the latest minor release of the selected major version at the time the Agent install command is run.

Reviewer: ```suggestion
   - Select the major version you want to use. The system installs the latest minor release of the selected major version.
```

Reviewer: Can you reword line 46 to be present tense, to conform to Datadog's style guide?

Author: I've updated the tense - I wasn't sure about the rule since I was trying to convey something that will happen in the future (at the time the command is run).

   ```

1. Find your language(s) and use the dropdown to either:
   - Select an exact SDK version, or

>    - Select the major version which uses the latest minor release available when the Agent installation command is run.

Reviewer: ```suggestion
   - Select the major version, which uses the latest minor release available when the Agent installation command is run.
```

>    - Select the major version which will use the latest minor release of the selected major version at the time the Agent install command is run.

Reviewer: ```suggestion
   - Select the major version you want to use. The system installs the latest minor release of the selected major version.
```

Reviewer: Can you reword line 67 to be present tense, to conform to Datadog's style guide?

Author: I've updated the tense - I wasn't sure about the rule since I was trying to convey something that will happen in the future (at the time the command is run).

  Enable products by setting environment variables directly in your application configuration. 

## Update SDK version

> The SDK version is fixed at the time the Agent installation command is run.

Reviewer: ```suggestion
The SDK version is fixed when you run the Agent installation command.
```

To update the SDK versions:

> 1. Re-run the Agent install command. This will also update the Agent to the latest version.

Reviewer: ```suggestion
1. Re-run the Agent install command. This command also updates the Agent to the latest version.
```
